index.js

```diff
@@ -32,4 +32,21 @@ module.exports = options => options ?
 	getPort(options).catch(() => getPort(Object.assign(options, {port: 0}))) :
 	getPort({port: 0});
 
+module.exports.makeRange = (from, to) => {
```

Comment:
I'm not sure the goal is to return an array of port numbers here. I think we should return the first available port from the input range.

Reply:
That was decided in original issue.

> I don't want to overload the `port` option with another value type, but maybe we could expose a `getPort.makeRange(from, to)` method for convenience. The benefit of having it as a method is that it's more flexible. You could combine ranges or filter out some specific ports from the range.
> 
> It should throw an error if the `from` argument is lower than 1024, as those are system/well-known ports.

Reply:
Yes, I know. I guess misinterpreted that. Thanks.